.changeset/brave-moons-repeat.md

```
@@ -0,0 +1,14 @@
+---
+'@mastra/hono': patch
+---
+
+Fix MCP Streamable HTTP client disconnects crashing the server with `ERR_INVALID_STATE`
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)